testing/integration/container_cmd_test.go

```diff
@@ -2,14 +2,16 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
-//go:build integration
+//go:build integration && linux
```

Comment:
Do we also want to run these tests on darwin?

Reply:
They're testing the container command, we only release Linux images, so I don't see the point of running them on Darwin.

If you look at the tests in the file, they all define only Linux:
```go
OS: []define.OS{
			{Type: define.Linux},
		},
```

Either way they will never run on Darwin. Having it only running on Linux also allows us to use some Linux specific code, which I had used in previous iterations of this test.

Reply:
I don't think we want to run the tests on Darwin but we don't want engineers on MacOS to be able to compile this file without jumping through hoops.

What problem is this solving? This seems like it is likely to waste MacOS users time when CI finds build errors they could have found locally.

Reply:
Indeed at the moment it solves no problem because I hardcoded `/tmp/elastic-agent/`, but there is a constant for it that is on a Unix-only file. I can remove it if you folks prefer.

Reply:
Yeah let's remove it, the intent isn't that these tests run on the host system, only that they compile there. Let's keep it that way as much as possible.